velox/docs/functions/spark/array.rst

```diff
@@ -75,6 +75,12 @@ Array Functions
         SELECT array_repeat(100, 0); -- []
         SELECT array_repeat(100, -1); -- []
 
+.. spark:function:: array_size(array(E)) -> integer
+    
+        Returns the size of the array. ::
```

Comment:
@acvictor Would you generate the docs locally and verify they render nicely?

Reply:
![image](https://github.com/facebookincubator/velox/assets/9670579/c97c95cf-c429-4376-a10b-12c4ae609123)
It looks good to me!

Reply:
Thank you for verifying.